### DISCUSSION TITLE
Blocked account

### DISCUSSION THREAD

**Discussion by ChineduCode** [vercel/community#3708] [Account Issue] [answered]
### Question

Am trying to signup with my github and it sent that am account has been blocked. Please what can I do?

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble creating a new account. Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username for the account. That will create a support case where you can privately discuss sensitive account information with the team. 🙇
